src/main/proto/wfa/measurement/reporting/v1alpha/metric_set.proto

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.reporting.v1alpha;
+
+import "google/api/field_behavior.proto";
+import "google/api/resource.proto";
+import "wfa/measurement/reporting/v1alpha/time_interval.proto";
+
+option java_package = "org.wfanet.measurement.reporting.v1alpha";
+option java_multiple_files = true;
+
+// A metric set using data from the Cross-Media Measurement System.
+message MetricSet {
```

Comment:
Is the idea that a Report will be N MetricSets? are we defining are Report elsewhere/later?